### ISSUE TITLE
Incompatibility with webonyx/graphql-php version 15.0

### ISSUE THREAD

**Issue by alleknalle** [overblog/GraphQLBundle#1085]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Version/Branch   | 0.15.1

`webonyx/graphql-php` released version 15.0 last friday and removed some Utils functions. 
Since there is no version constraint for `webonyx/graphql-php`, when using `overblog/graphql-bundle`, version 15.0 is used without the Utils, while `overblog/graphql-bundle` depends on the Utils function.
This is why we're getting the following error:
```
Uncaught PHP Exception Error: "Call to undefined method GraphQL\Utils\Utils::invariant()" at /var/www/vendor/overblog/graphql-bundle/src/Definition/Type/CustomScalarType.php line 72
```

**Comment by stevro** [comment]
This bug is happening for all v14.* and v15.* of overblog/GraphQLBundle as in composer.json we have "webonyx/graphql-php": ">=14.5" and it should be "webonyx/graphql-php": "^14.5"

**Comment by mcg-web** [comment]
Fixed in v0.15.2.
